code/data_structures/src/stack/stackUsingLinkedList/stackUsingLinkedList.cpp

```diff
@@ -0,0 +1,90 @@
+#include<iostream>
+#include<malloc.h>
+using namespace std;
+struct node
+{
+    int data;
+    struct node *next;
+};
+void Traverse_stack(struct node* top)
+{
+    struct node* temp=top;
+    cout<<"Current Stack->\n";
+    while(temp!=NULL)
```

Comment:
nullptr.

Reply:
both are same I think while(temp!=NULL) and while(!temp)

Reply:
They are, but don't use NULL in C++ if you want to be explicit, use nullptr instead.